### PR TITLE
fix: 🐛 delete defineProperty to recover doc.currentScript

### PR DESCRIPTION
**Pull request by stormslowly**
##### Checklist

- [x] `npm test` passes
- [ ] tests are included
- [ ] documentation is changed or added
- [x] commit message follows commit guidelines

##### Description of change

- close https://github.com/umijs/qiankun/issues/1689

### CODE REVIEW

File: src/sandbox/patchers/dynamicAppend/common.ts [hunk 1]
```diff
@@ -303,10 +303,18 @@ function getOverwrittenAppendChildOrInsertBefore(opts: {
               },
               success: () => {
                 manualInvokeElementOnLoad(element);
+                if(document.currentScript){
```
Comment (kuitos) [inline src/sandbox/patchers/dynamicAppend/common.ts:306]: 这里不需要判断可以无脑删吧？

Comment (stormslowly) [reply src/sandbox/patchers/dynamicAppend/common.ts:306]: isCurrentScriptConfigurable 有可能为 false  这个情况不知道能不能删

Comment (kuitos) [reply src/sandbox/patchers/dynamicAppend/common.ts]: 那应该是判断 isCurrentScriptConfigurable true 就能直接 delete 吧，而不是判断是否有值。

Comment (stormslowly) [reply src/sandbox/patchers/dynamicAppend/common.ts]: 换了种方式判断  配置过了，我就删； 没配置过就不动

**Review by kuitos — APPROVED** [review]
LGTM

**Pull request merged**